app/views/individual/incometax/subscription/claimEnrolment/ClaimEnrolmentConfirmation.scala.html

```diff
@@ -0,0 +1,73 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@import config.AppConfig
+@import views.html.templates.PrincipalMainTemplate
+@import views.html.helpers.injected.ButtonHelper
+@import views.html.helpers.injected.SignOutLink
+
+@this(
+        principalMainTemplate: PrincipalMainTemplate,
+        buttonHelper: ButtonHelper,
+        signOutLink: SignOutLink,
+        form: FormWithCSRF
+)
+@(postAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@principalMainTemplate(title = messages("claimConfirm.title")) {
+
+    <div id="confirmation-panel" class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
+        <h1 class="govuk-panel__title">@messages("claimConfirm.title")</h1>
+    </div>
+
+    @form(action = postAction) {
+    <section class="govuk-form-group" id="whatHappensNow">
+        <h2 class="govuk-heading-m">@messages("claimConfirm.heading")</h2>
+
+        <p class="govuk-body">@messages("claimConfirm.content.line1")
+
+        <ol id="actionList" class="govuk-list govuk-list--number">
+            <li>
+                <a class = "govuk-link" id="compatibleSoftware" href=@appConfig.softwareUrl target="_blank">
+                    @messages("claimConfirm.content.checklist.item1.link")
+                    @messages("claimConfirm.opens_in_a_new_tab")
+                </a>
+                @Html(messages(
+                    key = "claimConfirm.content.checklist.item1.suffix"
+                ))
+            </li>
+            <li>
+            @messages("claimConfirm.content.checklist.item2")
+            </li>
+            <li>
+            @messages("claimConfirm.content.checklist.item3")
+            </li>
+        </ol>
+
+        <p class="govuk-body">@Html(messages("claimConfirm.content.line2",
+                <a class="govuk-link" id="btaLink" href={appConfig.btaUrl} target="_blank">
```

Comment:
```suggestion
                <a class="govuk-link" id="btaLink" href={appConfig.incomeTaxViewChangeUrl} target="_blank">
```
We need to create a new appConfig value as above, this will require a change in frontendAppConfig and the application.conf files